README.md

```diff
@@ -17,14 +17,21 @@
 
 ## Usage
 
-> Some usage instructions
+### CRUD API
+To create a new user in the database:
+send POST request to domain/api/users with the user name in the body of the request with the name field (name: username)
 
-## Requirements
+To find a specific user by name:
+send GET request to domain/api/users/username
+
+To update a user avatar:
+send PUT request to domain/api/users with the user name in the body of the request with the name field (name: username)
+This assigns a random image to the user
 
-An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+To delete all reviews for a property listing:
+send DELETE request to domain/api/rentals/id where id is the number id of the listing
```

Comment:
Like the focused use of README - didn't try to rewrite, just stayed on point with putting in what had to be put in.